api/src/Classes/Models/VisitorModel.php

```diff
@@ -51,4 +51,59 @@ public function addVisitor($Name, $Company, $dateOfVisit, $timeOfSignIn, $signed
         $query->bindParam(':signedIn', $signedIn);
         return $query->execute();
     }
-}
+
+    /**
+     *  Getting visitors by name match.
+     *
+     * @param $Name
+     * @return array
+     */
+    public function getSignedInVisitorsByName($Name)
+    {
+        $query = $this->db->prepare(
+            "SELECT `id`, `Name`, `TimeOfSignIn` 
+                        FROM `visitors`
+                        WHERE `Name` = :Name
+                        AND `SignedIn` = 1 ");
```

Comment:
closing bracket should be on its own line? (like we would do with {} or [] if there were multiple lines?)